### ISSUE TITLE
Hard to follow the setup

### ISSUE THREAD

**Issue by bahunov** [SketchingDev/ivr-tester#92]
Hi @SketchingDev 

Could you please record tutorial or better documentation on how to get this working?

After downloading the repo, there's no mentioning of where to do npm install etc.
Where to create a sample test.js?

Easier tutorial please for dummies :)

Thanks!

**Comment by bahunov** [comment]
found solution:
```
import { googleSpeechToText } from "ivr-tester-transcriber-google-speech-to-text";
import { IvrTester, similarTo, press, hangUp, isAnything } from "ivr-tester";
import path from "path";
import { fileURLToPath } from 'url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);


const config = {
  transcriber: googleSpeechToText({ languageCode: "en-GB", useEnhanced: true }),
  twilioAuth: {
    accountSid: '',
    authToken: '',
  },
  recording: {
    transcript: {
      outputPath: path.join(__dirname, "../recordings"),
      includeResponse: true,
    },
  },
};

new IvrTester(config).run(
  { from: "+XXXXXX", to: "+XXXXXX" },
  {
    name: "Customer is provided a menu after their account number confirmed",
    steps: [
      {
        whenPrompt: isAnything(),
        then: press("112"),
        silenceAfterPrompt: 3000,
        timeout: 6000,
      },
      {
        whenPrompt: similarTo(
          "Finish"
        ),
        then: hangUp(),
        silenceAfterPrompt: 3000,
        timeout: 6000,
      },
    ],
  }
);
```

**Comment by SanaSajid96** [comment]
can u help m setting up i am facing same issue  of server has closed 
![Uploading help.PNG…]()

**Comment by SanaSajid96** [comment]
can u help m setting up i am facing same issue  of server has closed 
![help](https://github.com/SketchingDev/ivr-tester/assets/81260176/ed1c4e80-2ad2-4ec9-bbec-ea41b67a57dd)
